Describe process for contributing a new standard: (#96)
XRPLF/XRPL-Standards: xls-template.md

---
title: <The title is a few words, not a complete sentence>

> type: draft
> revision: 1

Author: ```suggestion
type: draft
```

description: <Description is one full (short) sentence>

> author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>

Author: ```suggestion
author: <a comma separated list of the author(s)>
# The format of each author should be one of the following:
# - Author's name, author's affiliation (organization or company) + GitHub username (in parenthesis)
# - Author's name, author's affiliation + email (in angle brackets)
#
# Example: FirstName LastName, OrganizationName (@GitHubUsername) , FirstName LastName, BarCompany <foo@bar.com>
```

> author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>

Author: Added.

requires: <XLS number(s)> # Only required when you reference an XLS in the `Specification` section. Otherwise, remove this field.
---

<!--

>   This is the suggested template for new XLS specifications. After you have filled in the requisite fields, please delete these comments.

Author: ```suggestion
  This is the suggested template for new XLS specifications. After you have filled in the requisite fields, please delete these comments.
  
  The requirements to sections depend on the type of proposal. For example, amendments require some information that may not be relevant for other kinds of proposals. Please adapt the template as appropriate.
```

  TODO: Remove this comment before submitting
-->

## Motivation

<!--
  This section is optional.

  The motivation section should include a description of any nontrivial problems the XLS solves. It should not describe how it solves those problems, unless it is not immediately obvious. It should not describe why the XLS should be made into a standard, unless it is not immediately obvious.

  With a few exceptions, external links are not necessary in this section. If you feel that a particular resource would demonstrate a compelling case for the XLS, then save it as a printer-friendly PDF, put it in the folder with this XLS, and link to that copy.

  TODO: Remove this comment before submitting
-->

## Specification

<!--
  The Specification section should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations.

  It is recommended to follow RFC 2119 and RFC 8170. Do not remove the key word definitions if RFC 2119 and RFC 8170 are followed.

  TODO: Remove this comment before submitting
-->

> 
> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.

Author: ```suggestion

The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.

<!--
The following is an example of how you can document new object types and fields:

#### The **`<object name>`** object

<High level overview, explaining the object>

##### Fields

<Any explanatory text about the fields overall>

---

| Field Name        | Required?        |  JSON Type      | Internal Type     |
|-------------------|:----------------:|:---------------:|:-----------------:|
| `<field name>` | :heavy_check_mark: | `<string, number, object, array, boolean>` | `<UINT128, UINT160, UINT256, ...>` |

<Any explanatory text about specific fields>

###### Flags

> | Flag Name            | Flag Value  | Description |
>|:---------------------:|:-----------:|:------------|
>| `lsf<flag name>` | `0x0001`| <flag description> |

<Any explanatory text about specific flags>
-->
```

  If the reference implementation is too large to reasonably be included inline, then consider adding it as one or more files in the folder with this XLS. External links are discouraged.

  TODO: Remove this comment before submitting
-->

## Security Considerations

<!--
  All XLS documents must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks, and can be used throughout the lifecycle of the proposal. For example, include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks, and how they are being addressed. Submissions missing the "Security Considerations" section may be rejected.

  The current placeholder is acceptable for a draft.

  TODO: Remove this comment before submitting
-->

Needs discussion.

> 

Author: Added in 2e63068ff40c61457c81c47810200225bd2affc1

---
title: <The title is a few words, not a complete sentence>
type: draft
revision: 1
description: <Description is one full (short) sentence>

> author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>

Reviewer: Can we possibly consider a way to list out authors related to specific revision numbers, like a list of revision # and the authors?  I think that would be suitable for the discussion of clawback, where @nbougalis was the author of revision 1 but someone else may be the author for revision 2

Author: Sure, what would you suggest?

In my opinion, it doesn't matter and isn't necessary. Readers who care about who authored individual revisions can simply look back at the Discussion/PR/commit history.

Reviewer: I don't have a specific suggestion, but maybe perhaps multiple rows of 
```
---
revision: 2
author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
---
revision: 1
author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
---
```

Author: Small updates do not require numbered versions, so the revision numbers have been removed. With git, it's easy to view the history of any file.

  If the reference implementation is too large to reasonably be included inline, then consider adding it as one or more files in the folder with this XLS. External links are discouraged.

  TODO: Remove this comment before submitting
-->

## Security Considerations

<!--
  All XLS documents must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks, and can be used throughout the lifecycle of the proposal. For example, include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks, and how they are being addressed. Submissions missing the "Security Considerations" section may be rejected.

  The current placeholder is acceptable for a draft.

  TODO: Remove this comment before submitting
-->

Needs discussion.

## Copyright

> Copyright and related rights waived via [CC0](../LICENSE.md).

Reviewer: should this be necessary for an open-source repo under MIT License?

> ## Copyright
> 
> Copyright and related rights waived via [CC0](../LICENSE.md).

Author: ```suggestion
```

---
title: <The title is a few words, not a complete sentence>
type: draft
description: <Description is one full (short) sentence>
author: <a comma separated list of the author(s)>
# The format of each author should be one of the following:
# - Author's name, author's affiliation (organization or company) + GitHub username (in parenthesis)
# - Author's name, author's affiliation + email (in angle brackets)
#
# Example: FirstName LastName, OrganizationName (@GitHubUsername) , FirstName LastName, BarCompany <foo@bar.com>

> requires: <XLS number(s)> # Only required when you reference an XLS in the `Specification` section. Otherwise, remove this field.

Reviewer 2: LGTM!
